src/utils.ts

```diff
@@ -150,4 +150,17 @@ export function isDynamicPattern(pattern: string, options?: { caseSensitiveMatch
 export function log(...tasks: unknown[]): void {
   console.log(`[tinyglobby ${new Date().toLocaleTimeString('es')}]`, ...tasks);
 }
+export function coloredLog(msg: string, type: 'matched' | 'skipped' | 'crawling'): void {
+  const colors = {
+    matched: '\x1b[32m',
+    skipped: '\x1b[90m',
+    crawling: '\x1b[34m'
+  };
+  const color = colors[type as keyof typeof colors];
+  if (color) {
+    console.log(`${color}%s\x1b[0m`, `[tinyglobby ${new Date().toLocaleTimeString('es')}]${msg}`);
```

Comment:
hardcoding it to spanish?

Reply:
that one isn't changed on this pr and comes from main, it's for hardcoding time to be in the hh:mm:ss format, couldn't find a "international" locale that did this

Reply:
Ah, makes sense l. Could be worth a comment

Reply:
Swedish is quite often used for this kinda thing but that's a bit risky though, I've opened an issue in NodeJS a while ago for date string not respecting 2 digits with 0 padding in Node 22. Then a Node maintainer replied with this [comment](https://github.com/nodejs/node/issues/51090#issuecomment-1852036829)

> Data changes constantly. Look at https://www.unicode.org/cldr/charts/44/delta/index.html and pick any locale. `toLocaleString()` is completely inappropriate as an API to expect a constant format. It should display what's correct for Swedish, which might change over time.

It's usually safer to use `new Date(date).toISOString()` even though it's UTC TZ